static/app/components/profiling/flamegraph/flamegraphSpans.tsx

```diff
@@ -367,6 +392,108 @@ export function FlamegraphSpans({
     };
   });
 
+  const onCopyDescription = useCallback(() => {
+    const value = hoveredNodeOnContextMenuOpen.current?.node.span.description ?? '';
+    if (!value) {
+      addErrorMessage(t('Event description value is empty.'));
+      return;
+    }
+
+    navigator.clipboard
+      .writeText(value)
+      .then(() => {
+        addSuccessMessage(t('Description copied to clipboard'));
+      })
+      .catch(() => {
+        addErrorMessage(t('Failed to copy description to clipboard'));
+      });
+  }, [hoveredNodeOnContextMenuOpen]);
+
+  const onCopyOperation = useCallback(() => {
+    const value = hoveredNodeOnContextMenuOpen.current?.node.span.op ?? '';
+    if (!value) {
+      addErrorMessage(t('Event operation value is empty.'));
+      return;
+    }
+
+    navigator.clipboard
+      .writeText(value)
+      .then(() => {
+        addSuccessMessage(t('Operation copied to clipboard'));
+      })
+      .catch(() => {
+        addErrorMessage(t('Failed to copy operation to clipboard'));
+      });
+  }, [hoveredNodeOnContextMenuOpen]);
+
+  const onCopyEventId = useCallback(() => {
+    const value =
+      hoveredNodeOnContextMenuOpen.current?.node.span.event_id ??
+      hoveredNodeOnContextMenuOpen.current?.node.span.span_id ??
+      '';
+    if (!value) {
+      addErrorMessage(t('Event ID value is empty.'));
+      return;
+    }
+
+    navigator.clipboard
+      .writeText(value)
+      .then(() => {
+        addSuccessMessage(t('Event ID copied to clipboard'));
+      })
+      .catch(() => {
+        addErrorMessage(t('Failed to copy event ID to clipboard'));
+      });
+  }, [hoveredNodeOnContextMenuOpen]);
+
+  const onOpenInTraceView = useCallback(() => {
+    const node = hoveredNodeOnContextMenuOpen.current?.node;
+    if (!node) {
+      addErrorMessage(t('No event ID or span ID found'));
+      return;
+    }
+
+    const nodePath: Record<string, string> = {};
+
+    if (node.span.op === 'transaction' && node.span.event_id) {
+      // If the user clicks on a transaction, we can directly use the event_id
+      nodePath.eventId = node.span.event_id;
+    } else if (node.span.span_id) {
+      // If the user clicks on a span, we need to traverse up the tree to find the transaction so that
+      // the trace view knows which transaction to load and what span to point to.
+      nodePath.spanId = node.span.span_id;
+      let parent = node.parent;
+      while (parent) {
+        if (parent.span.op === 'transaction' && parent.span.event_id) {
+          nodePath.eventId = parent.span.event_id;
+          break;
+        }
+        parent = parent.parent;
+      }
```

Comment:
Do we ever have more than 1 transaction on this page? This is more robust but perhaps unneeded.

Reply:
No, we dont, but we always want to just look for the first parent txn because we have the trace id and its enough for us to link to it